### ISSUE TITLE
Add localization platfrom - Hosted Weblate

### ISSUE THREAD

**Issue by inson1** [VueTubeApp/VueTube#671]
### Describe your suggested feature

It would be easier than creating PRs
Second option is Crowdin, but isnt open source

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by ngocanhtve** [comment]
Yes, but it's less efficient than a PR. Editing a translation on a PC with notepad++, for example, gives us a more comprehensive view of the entire translation, unifying the style throughout the text.

For example, if I have 200 lines of text to translate, on weblate I cannot complete it in a working day, but if I do it on notepad++ and then make a PR, it only takes me a couple of hours.

**Comment by inson1** [comment]
@ngocanhtve I dont feel more productive with PRs tbh, but that doesnt matter :) 
I'm not saying ban PRs with translations :) this is just request for another option :)

**Comment by ngocanhtve** [comment]
@inson1 Yeah, you're right. I'm also not saying there shouldn't be another option, I'm quite in favor of weblate.

But I'm just pointing out the outstanding performance advantages of a PR.
